sui/src/api.rs

```diff
@@ -137,56 +128,33 @@ pub trait RpcGateway {
         digest: TransactionDigest,
     ) -> RpcResult<TransactionEffectsResponse>;
 
-    /// Low level API to get object info. Client Applications should prefer to use
-    /// `get_object_typed_info` instead.
-    #[method(name = "getObjectInfoRaw")]
-    async fn get_object_info(&self, object_id: ObjectID) -> RpcResult<ObjectRead>;
-}
-
-#[serde_as]
-#[derive(Serialize, Deserialize, JsonSchema)]
-pub struct SignedTransaction {
-    #[schemars(with = "json_schema::Base64")]
-    #[serde_as(as = "base64::Base64")]
-    pub tx_bytes: Vec<u8>,
-    #[schemars(with = "json_schema::Base64")]
-    #[serde_as(as = "base64::Base64")]
-    pub signature: Vec<u8>,
-    #[schemars(with = "json_schema::Base64")]
-    #[serde_as(as = "base64::Base64")]
-    pub pub_key: Vec<u8>,
-}
-
-impl SignedTransaction {
-    pub fn new(tx_bytes: Vec<u8>, signature: crypto::Signature) -> Self {
-        Self {
-            tx_bytes,
-            signature: signature.signature_bytes().to_vec(),
-            pub_key: signature.public_key_bytes().to_vec(),
-        }
-    }
+    /// Return the object information for a specified object
+    #[method(name = "getObjectInfo")]
+    async fn get_object_info(&self, object_id: ObjectID) -> RpcResult<SuiObjectRead>;
 }
 
 #[serde_as]
 #[derive(Serialize, Deserialize, JsonSchema)]
 pub struct TransactionBytes {
-    #[schemars(with = "json_schema::Base64")]
-    #[serde_as(as = "base64::Base64")]
-    pub tx_bytes: Vec<u8>,
-    pub gas: ObjectRef,
-    pub input_objects: Vec<InputObjectKind>,
+    pub tx_bytes: Base64,
+    pub gas: SuiObjectRef,
+    pub input_objects: Vec<SuiInputObjectKind>,
```

Comment:
nit: we can consider deserializing these fields to camelCase as well(or define a new one if it affects other places) to be consistent with the rest of the API

Reply:
sure, I will add `SuiInputObjectKind `